nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/test/java/org/nuxeo/ecm/core/storage/sql/TestNXQLQueryMaker.java

```diff
@@ -45,4 +48,19 @@ public void testSimpleXPath() throws Exception {
         assertEquals("abc/def1", simpleXPath("abc/def1"));
     }
 
+    @Test
+    public void testNowPeriodAndDuration() throws Exception {
+        DateTime dateTime = new DateTime(2001, 2, 3, 4, 5, 6, 7, ISOChronology.getInstanceUTC());
+        assertEquals("2001-02-03T04:05:06.007Z", dateTime.toString());
+        //
```

Comment:
?

Reply:
Just spacing before the actual tests

Reply:
Yep, but why do you need the `//`?

Reply:
A blank line was too much whitespace in my eye 😀